notification/static/notification/css/notification.css

```diff
@@ -1,5 +1,22 @@
 .card-badge {
     position: absolute;
-    right: 25px;
+    right: -25px;
     bottom: 35%;
```

Comment:
I suggest to make accordion title centred vertically by setting margin-bottom: 0; and adjusting card-badge bottom:0; to make it centred too.

Reply:
I'll check, thanks!

Reply:
It should be solved now. Could you have a look?